module/mempool/stdmap/backdata/herocache/cache.go

```diff
@@ -177,21 +177,31 @@ func (c Cache) All() map[flow.Identifier]flow.Entity {
 	defer c.logTelemetry()
 
 	all := make(map[flow.Identifier]flow.Entity)
```

Comment:
maybe you can pre-allocate space in the map, by adding a parameter to make with the length of 'entities.All()' ?

Reply:
https://github.com/onflow/flow-go/pull/1852/commits/d62f7c79438a16f96d13781557064140c6e154a5